psi4/driver/procrouting/proc.py

```diff
@@ -53,6 +53,7 @@
 from . import dft
 from . import mcscf
 from . import response
+from . import solvent
```

Comment:
note for a second review pass: need to find a spot (scf_helper?) where we look if `PE` is on and if so, set c1/noreorient so user doesn't need to handle this.

Reply:
I'll try to implement a clean way of doing this... the problem is that PE needs information about the molecule and basis set to construct the `MintsHelper` downstream... So one would have to do it [here](https://github.com/psi4/psi4/blob/083fb200b45aa3e0390b934b23627d83d9abf3bc/psi4/driver/procrouting/proc.py#L1500), where the molecule/basisset are already transformed to C1.

Reply:
Okay, I give up on this for today...

Reply:
I guess the only way to do this is through the user input. The molecule must not be moved or rotated... and there is no way of knowing if PE is enabled at the moment the molecule is constructed. Any suggestions?

Reply:
Unless PE uses are peculiar, don’t need to know c1ness at mol creation time, only need to set before computation time. Mol class stores original and applies transformations.

Reply:
I'm giving up on this... probably you can help, @loriab ☹️ 
I played around with code from [here](https://github.com/psi4/psi4/blob/a8b9036e49d01f385fa3ccfb4b3eb48c13785899/psi4/driver/procrouting/proc.py#L1461), but could not make it work after all...